### ISSUE TITLE
Node 6.0.0 error

### ISSUE THREAD

**Issue by DanielRamosAcosta** [fisherman/fin#14]
When I use version 6.0.0 it gives an error, but Node and NPM works. I don't know if this matters, but I reported it anyways:

```fish
 ~> fin use 6.0.0                                                                                                 mié 27 abr 2016 12:34:07 WEST
Downloading <https://nodejs.org/dist/v6.0.0/node-v6.0.0-linux-x64.tar.gz>
######################################################################## 100,0%
fish: Unknown command 'cd /home/daniel/.cache/fin/versions/6.0.0'
fish:         command cd '/home/daniel/.cache/fin/versions/6.0.0'
```

**Comment by DanielRamosAcosta** [comment]
Also, global packages aren't being added to `$PATH`

**Comment by bucaran** [comment]
@DanielRamosAcosta 

Hey, can you give me more information? OS name, architecture, etc. Also, look at the code in `__fin_run_bin_as` and see if you sport anything strange. 


> Also, global packages aren't being added to $PATH

Looks like a different issue. Can you create a new issue for this?

**Comment by DanielRamosAcosta** [comment]
* Distributor ID: `Ubuntu`
* Description: `Ubuntu 16.04 LTS`
* Release: `16.04`
* Codename: `xenial`
* Linux version: `Linux 4.4.0-21-generic x86_64`
* Fish version: `2.3b1`
* Fisherman version: `2.1.13`

I'll post here my `__fin_run_bin_as.fish` here, becouse I wouldn't recognize an error...

**Comment by bucaran** [comment]
@DanielRamosAcosta Thanks. I am working on fixing #15 first as that has higher priority, but can you:

```
uname -a
```

Thanks.

**Comment by DanielRamosAcosta** [comment]
Here:
`Linux daniel-lenovo 4.4.0-21-generic #37-Ubuntu SMP Mon Apr 18 18:33:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by bucaran** [comment]
Thanks. I'll get back to this after #15.

**Comment by bucaran** [comment]
Ping @DanielRamosAcosta.

fin supports global modules now, works just fine when installing node versions via  `fin [use] <version>`. 

When you use a `.finrc` file, the selection only lasts as long as the duration of the command:

```
echo latest
node args # fin use X.X.X; run node with args; fin rm X.X.X
```

So, `npm i -g some_stuff` installs correctly using the version in the `.finrc`, but it's soon removed after the npm exits since `fin` restores the state soon after. Which might be what you want. If that's not what you want, then you can simply `fin X.X.X` and it'll work as expected, plus it's so much faster.

<h1></h2>


With regards to the other issue, could you please try again with fin@1.2.0 and let me know if the problem persists?

**Comment by DanielRamosAcosta** [comment]
I'm don't understanding the purpose of the `.finrc` file... I just updated fin to `1.3.0` and error seems to persist, and to be for other versions not just `6.0.0`. I'll comment about global packages in #15.
![use](https://cloud.githubusercontent.com/assets/11427028/14864613/91ecf1f2-0cb2-11e6-8920-94700cbab0dc.png)

**Comment by bucaran** [comment]
@DanielRamosAcosta The comment about the .finrc file is just in case you were using them, if you are not using a `.finrc` file, then don't worry about it :)

It seems you are still getting the same error. This is really odd. Is `cd` not a command in your system?

Try 

```
command cd
```

**Comment by bucaran** [comment]
Maybe the `Unknown command` error is fixed now:

* https://github.com/fisherman/fin/commit/65eda58ee8f2a7cea2d74c10968d3dfe2f8914df

Can you check again?

And as for the global packages issue, I already replied here: https://github.com/fisherman/fin/issues/15#issuecomment-215284959

**Comment by DanielRamosAcosta** [comment]
Yay! now is fixed! And yes, `command cd` gives me an error. What is `command` for?

Thanks a lot @bucaran , fin and fisher is awesome!

**Comment by bucaran** [comment]
@DanielRamosAcosta 

`command <program>` is used to a run *<program>* in your system, even if there is a function in your session with the same name.

So, is the error gone?

**Comment by DanielRamosAcosta** [comment]
Yes!

**Comment by bucaran** [comment]
Yay!
